ts/receiver/contentMessage.ts

```diff
@@ -0,0 +1,526 @@
+import { EnvelopePlus } from './types';
+import { handleDataMessage } from './dataMessage';
+import { getEnvelopeId } from './common';
+
+import { removeFromCache, updateCache } from './cache';
+import { SignalService } from '../protobuf';
+import { toNumber } from 'lodash';
+import * as libsession from '../session';
+import { handleSessionRequestMessage } from './sessionHandling';
+import { handlePairingAuthorisationMessage } from './multidevice';
+import {
+  MediumGroupRequestKeysMessage,
+  ReceiptMessage,
+} from '../session/messages/outgoing';
+import { MultiDeviceProtocol, SessionProtocol } from '../session/protocols';
+import { PubKey } from '../session/types';
+
+import { handleSyncMessage } from './syncMessages';
+import { onError } from './errors';
+
+export async function handleContentMessage(envelope: EnvelopePlus) {
+  const plaintext = await decrypt(envelope, envelope.content);
+
+  if (!plaintext) {
+    window.log.warn('handleContentMessage: plaintext was falsey');
+    return;
+  } else if (plaintext instanceof ArrayBuffer && plaintext.byteLength === 0) {
+    return;
+  }
+  await innerHandleContentMessage(envelope, plaintext);
+}
+
+async function decryptForMediumGroup(
+  envelope: EnvelopePlus,
+  ciphertextObj: ArrayBuffer
+) {
+  const { dcodeIO, libloki } = window;
```

Comment:
`ByteBuffer` can be directly imported in ts.
```
import ByteBuffer from 'bytebuffer';
```

Reply:
updated